src/App.js

```diff
@@ -9,7 +11,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div className="app">
+    <h1>React Article</h1>
```

Comment:
Page ususally has to have only one `h1` element. So, this header can be safely deleted.